src/albums_test.go

```diff
@@ -39,3 +40,46 @@ func TestLoadPhotoInAlbum(t *testing.T) {
 		}
 	}
 }
+
+// Validate when we query the album with a date we have our photo
+func TestGetLstPhotosForWeek(t *testing.T) {
```

Comment:
I think I can do more validation but the time run and I would like to have the web interface available. 
I already know I will not be able to finish :) , so more validation should be performed but at least we have some validation now.